tests/unit/s2n_random_test.c

```diff
@@ -471,10 +465,9 @@ static S2N_RESULT s2n_clone_tests(
     char process_child_stack[PROCESS_CHILD_STACK_SIZE];
     EXPECT_NOT_NULL(process_child_stack);
 
-    struct random_communication ipc =
-        { .s2n_get_random_data_cb_1 = s2n_get_random_data_cb,
-          .s2n_get_random_data_cb_2 = s2n_get_random_data_cb_clone,
-          .pipes = (int *) pipes };
+    struct random_communication ipc = { .s2n_get_random_data_cb_1 = s2n_get_random_data_cb,
+        .s2n_get_random_data_cb_2 = s2n_get_random_data_cb_clone,
+        .pipes = (int *) pipes };
```

Comment:
Nit: might want to clean this up a bit. Will clang-format accept
```suggestion
    struct random_communication ipc = {
        .s2n_get_random_data_cb_1 = s2n_get_random_data_cb,
        .s2n_get_random_data_cb_2 = s2n_get_random_data_cb_clone,
        .pipes = (int *) pipes
    };
```

Reply:
It does. This is changed.